guide/05-working-with-the-spatially-enabled-dataframe/part2_data_io_reading_data.ipynb

```diff
@@ -0,0 +1,5229 @@
+{
```

Comment:
<h4>Using the <code>from_table()</code> method </h4><p> </p><p>A <code>SeDF</code> can be created from local non-spatial data using the <code><a href="https://developers.arcgis.com/python/api-reference/arcgis.features.toc.html?highlight=from_feather#arcgis.features.GeoAccessor.from_table" target="_blank">from_table()</a></code> method. The method can read a csv file (in any environment) or a table stored in a File Geodatabase (with ArcPy only).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='138'/>